crates/wasm/Cargo.toml

```diff
@@ -17,19 +17,18 @@ homepage = "https://fission.codes"
 authors = ["The Fission Authors"]
 
 [dependencies]
-libipld = "0.14.0"
-wnfs = { path = "../fs", version = "0.1.8" }
-wasm-bindgen = { version = "0.2", optional = true, features = ["serde-serialize"] }
-wasm-bindgen-futures = { version = "0.4", optional = true }
-js-sys = { version = "0.3", optional = true }
-web-sys = { version = "0.3", optional = true }
-chrono = { version = "0.4", features = ["wasmbind"] }
 anyhow = "1.0"
 async-trait = "0.1"
+cfg-if = "1.0"
+chrono = { version = "0.4", default-features = false, features = ["clock", "std", "wasmbind"] }
 console_error_panic_hook = { version = "0.1", optional = true }
-wee_alloc = { version = "0.4", optional = true }
```

Comment:
Turns out wee-alloc hasn't brought much improvement to our wasm size or we've been using it wrong.

Reply:
It's essentially dead anyway. https://github.com/rustwasm/wee_alloc/issues/85 + https://github.com/rustsec/advisory-db/issues/1394. We can look at other allocators tho, of interest.